docs/src/ComponentsPage.js

```diff
@@ -733,6 +733,12 @@ const ComponentsPage = React.createClass({
                   <p>Set the <code>header</code> prop to create a structured item, with a heading and a body area.</p>
                   <ReactPlayground codeText={Samples.ListGroupHeader} />
 
+                  <h3><Anchor id="listgroup-with-custom-children">With custom component children</Anchor></h3>
```

Comment:
Not sure it adds a ton, but I thought it better to over document then under. I can remove.